### ISSUE TITLE
A11y_AzureDataStudio_BigDataClusterOverviewPage_AI4D: Incorrect control type is given for "Checked" and "Warning " image.

### ISSUE THREAD

**Issue by 12-shweta** [microsoft/azuredatastudio#7669]
"[Check out Accessibility Insights!](https://accessibilityinsights.io/)- Identify accessibility bugs before check-in and make bug fixing faster and easier."
​
GitHubTags:#A11y_AzureDataStudio;#A11yMAS;#A11yTCS;#WCAG1.3.1;#WCAG4.1.2;DesktopApp;#NVDA;#JAWS;#SQL Azure Data Studio;#AI4D;#Win32

**Environment Details:**
Application Name: Azure Data Studio
Version: 1.12.0-insider (user setup)
Commit: 6582debd7388b1be512fc3a9b956d3e54f6115e7
Date: 2019-10-03T08:56:27.454Z
VS Code: 1.38.0
Electron: 4.2.10
Chrome: 69.0.3497.128
Node.js: 10.11.0
V8: 6.9.427.31-electron.0
OS: Windows_NT x64 10.0.18362
NVDA Version: 2019.2
JAWS Version:2019.1907.42 ILM

**Additional Details:**
Tool Name : AI4D

**Steps to Reproduce:**
1. Install and Launch the application the azure data studio application.
2. Open the “Connections” viewlet (CTRL + SHIFT + D)
3. Navigate to the “SQL Server Big Data Clusters” section and expand it
4. Navigate to the "Add SQL Server big data cluster controller…" node and click on it.
5. Fill in the connection information for the BDC Controller Endpoint
6. Click “Add”
7. Navigate to the controller node in the “SQL Server Big Data Clusters” view and right click on it, then select “Manage” to open the dashboard
8. "Big Data Cluster Overview page " should open.
9. Open AI4D.
10. Check the control type given for "Checked" and "Warning " image.

**Actual:**
Incorrect control type is given for "Checked" and "Warning " image.
Given control type is "text".

**Similar issue is also observed with JAWS .**
**Similar issue is also observed with following screens:
1)Control tab**
1. Install and Launch the application the azure data studio application.
2. Open the “Connections” viewlet (CTRL + SHIFT + D)
3. Navigate to the “SQL Server Big Data Clusters” section and expand it
4. Navigate to the "Add SQL Server big data cluster controller…" node and click on it.
5. Fill in the connection information for the BDC Controller Endpoint
6. Click “Add”
7. Navigate to the controller node in the “SQL Server Big Data Clusters” view and right click on it, then select “Manage” to open the dashboard->click on the "control tab " in the left nav bar-> navigate to the "Metricsdc" tab.

 
**Expected:**
Correct control type should be provided for "Checked" and "Warning " image.
It should be image and  screen reader should announce the control type provided for it

**User Impact:**
If proper control type is not provided then screen reader users will get confused with the control..

**Attachment for Reference:**
![7669_A11y_AzureDataStudio_BigDataClusterOverviewPage_AI4D](https://user-images.githubusercontent.com/54656883/66646838-ea0a8500-ec44-11e9-80ab-a87155c82dfb.png)

Does this issue occur when all extensions are disabled?: Yes

**Comment by 12-shweta** [comment]
Verified the bug in below environment :
Version: 1.15.0-insider (user setup)
Commit: 041f34b
Date: 2020-01-08T05:36:19.161Z
VS Code: 1.38.0
Electron: 6.1.6
Chrome: 76.0.3809.146
Node.js: 12.4.0
V8: 7.6.303.31-electron.0
OS: Windows_NT x64 10.0.18363
Issue is fixed and does not reproduce .
![7669_Closed](https://user-images.githubusercontent.com/54656883/72725366-9ac0e280-3bab-11ea-8253-2005391cb61b.png)

**Comment by fsteffi** [comment]
#Azure Data Studio - New Features_win32;

**Comment by Rama8345** [comment]
A11ySev2

**Comment by Samhitha66** [comment]
#Closed; #Regressed:07-14-23;
